.github/workflows/ci.yaml

```diff
@@ -19,7 +19,7 @@ jobs:
       matrix:
         os: [ubuntu-latest, windows-latest, macOS-latest]
         rust: [nightly]
-        backend: [curl-client, h1-client, hyper-client]
+        backend: [curl-client, h1-client, hyper-client, h1-client-rustls]
```

Comment:
I'd rather test this in http-client, to avoid ballooning the feature matrix too much here

Reply:
Sure, I'll remove it.